cetmix_tower_server/models/cx_tower_server.py

```
@@ -1550,57 +1552,58 @@
-                    command_code.replace("\\", "").replace("\n", "").split(separator)
-                )
-
-                # Sudo with password expects a list of commands
-                result = [f"{sudo_prefix} {cmd.strip()}" for cmd in result]
-
-                # Merge back into a single command is sudo is without password
-                if sudo == "n":
-                    result = f" {separator} ".join(result)
-            else:
-                # Single command only
+            # If No Split for sudo flag is active, we don't separate the commande even we have &&:
```

Ruff (0.8.2)

Line too long (98 > 88)

(E501)